SPECS/polkit/polkit.spec

```diff
@@ -15,12 +15,13 @@ BuildRequires:     gobject-introspection-devel
 BuildRequires:     intltool >= 0.40.0
 BuildRequires:     meson
 BuildRequires:     pam-devel
-BuildRequires:     systemd-devel
+BuildRequires:     systemd-bootstrap-devel
 Requires:          duktape
 Requires:          expat
 Requires:          glib
 Requires:          pam
-Requires:          systemd
+# Our build tooling cannot handle this
+#Requires:          systemd
```

Comment:
Same question here as in `iptables.spec`.

Reply:
same circular *runtime* dep problem. I've no idea why the toolkit fails due to *runtime* "circular" deps when building a package.